StPetersburg/RNBefns7.xml

```diff
@@ -24,136 +24,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <condition key="good"/>
-                
                      </supportDesc>
-              
-              
-              
+
                      <layoutDesc>
                         <layout columns="2" writtenLines="14">
-                           <locus from="1r" to="84"/>                        
-                        </layout>   
+                           <locus from="1r" to="84"/>
+                        </layout>
                      </layoutDesc>
                   </objectDesc>
-            
-            
-            
+
                   <handDesc>
                      <handNote xml:id="h1" script="Ethiopic">
                         <date notBefore="1500" notAfter="1600"/>
-                        <desc>The handwriting is defined by <persName ref="PRS9602Turaev"/> as "fine"</desc> 
+                        <desc>The handwriting is described by <persName ref="PRS9602Turaev"/> as "fine".</desc>
                      </handNote>
-                  </handDesc> 
-            
-            
+                  </handDesc>
+
+
                   <decoDesc>
                      <decoNote type="band" xml:id="d1">
                         <locus target="#1r"/>
-                        <desc> Polichrom ornamental band</desc>
-                        <note>Presence of other ornamental bands is indicated in the catalogue, but no foli numbers are provided </note>
-                     </decoNote>             
+                        <desc>Polichrom ornamental band.</desc>
+                        <note>Presence of other ornamental bands is indicated in the catalogue,
+                          but no folio numbers are provided.</note>
+                     </decoNote>
                   </decoDesc>
-            
-                  <additions>
-                     <list>
-                        <item xml:id="a1">
-                           <desc type="MagicText"> </desc>
-                        </item>
-                     </list>
-                  </additions>               
-            
-            
+
+                  <additions/>
+
+
                   <bindingDesc>
                      <binding contemporary="false">
-                        <decoNote xml:id="b1"> Binding not Ethiopian, made of tooled Morocco leather. 
-                  One pair of sewing stations can be assumed (not indicated in the catalogue)</decoNote> 
-                        <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
-                        <decoNote xml:id="b3" type="bindingMaterial">
-                           <material key="leather"/>
+                        <decoNote xml:id="b1">Binding non-Ethiopian, made of tooled Morocco leather.</decoNote>
+                <decoNote xml:id="b2">Probably one pair of sewing stations.</decoNote>
```

Comment:
where is b3?

Reply:
introduced

Reply:
please make a further commit with this updates to this branch